internal/pkg/levee/levee_ear_test.go

```diff
@@ -0,0 +1,69 @@
+	}
+
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/arguments")
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/basictypes")
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/binop")
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/booleans")
+	// analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/callorder")  // TODO: flow-sensitive
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/closures")
+	// analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/collections") // TODO: map key
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/declarations")
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/eface")
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/embedding")
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/excludedpackage")
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/extracts")
+	// analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/fields")  // TODO: FP has been fixed?
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/includedpackage")
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/inlining")
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/loops")
+
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/namedreturn")
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/panic")
+
+	// analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/phi")  // TODO: flow sensitive
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/pointers")
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/position")
+	// analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/propagation") // TODO: flow sensitive
+	// analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/propagation/builtins.go") // TODO: flow sensitive
+
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/receivers")
+	analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/recover")
+	// analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/sanitization")  // TODO: santizers are not handled yet
+	// analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/sinks")  // TODO
+	// analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/store")  // TODO: flow sensitive
+	// analysistest.Run(t, dataDir, Analyzer, "./src/levee_analysistest/example/tests/structlit")  // TODO: NP have been fixed?
```

Comment:
Yep, it looks like EAR can handle all of these cases.

Reply:
There a small number of either FPs or TPs in these tests. Leave as future work.